src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs

```diff
@@ -221,12 +221,12 @@ private bool CanJoinTally()
     {
         return Tally?.State == TallyState.PendingGuardiansJoin &&
             !CurrentUserJoinedAlready() &&
-            !AuthenticationService.IsAdmin;
+            !IsAdmin;

```

Comment:
This was a lot of why the Auth service was pulled into this VM. Can we remove it?